subprojects/docs/src/docs/userguide/basePlugin.adoc

```diff
@@ -0,0 +1,90 @@
+// Copyright 2018 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[base_plugin]]
+== The Base Plugin
+
+The Base Plugin provides features that are common to most builds and adds a structure to the build that promotes consistency in how they are run. Its most significant contribution is a set of <<sec:base_tasks,'lifecycle' tasks>> that act as an umbrella for the more specific tasks provided by other plugins and build authors.
```

Comment:
I'd probably put an emphasis on the term "convention" here. You description basically says that but I wouldn't call it a feature I think.

Reply:
It provides the `clean` and `upload*` tasks as well, hence the use of features. It's not just a set of conventions. That said, the conventions are its main focus I guess.

Reply:
- [ ] Improve on 'features'

Reply:
It's definitely a combination of both: tasks and conventions. For some reason "feature" sounds like something bigger but that may one by my perception.